03_letters/willshen8/letters.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func letters(s string) map[rune]int {
+	letterMap := make(map[rune]int)
+
+	for _, value := range s {
+		if letterFrequency, found := letterMap[value]; found {
+			letterMap[value] = letterFrequency + 1
+		} else {
+			letterMap[value] = 1
+		}
+	}
+	return letterMap
+}
+
+func sortLetters(m map[rune]int) []string {
+	frequencySlice := []string{}
+
+	for k, v := range m {
+		convertedKeyInStringFormat := string(k)
```

Comment:
Consider shorter names when in small blocks

Reply:
Thanks done.